cmd/abad/cli/cli.go

```diff
@@ -34,21 +39,19 @@ func (c *Cli) ReadEval() {
 	bio := bufio.NewReader(c.in)
 	line, err := bio.ReadString('\n')
 	if err != nil {
-		c.errorf(err)
+		c.error(err)
 		return
 	}
 
-	if len(line) > 0 {
-		line = line[:len(line)-1]
-	}
+	line = trimnl(line)
 
 	obj, err := c.js.Eval(line)
 	if err != nil {
-		c.errorf(err)
+		c.error(err)
 		return
 	}
 
-	fmt.Fprintf(c.out, "< %s\n", obj)
+	fmt.Fprintf(c.out, "< %s\n", obj.ToString().String())
```

Comment:
hmm it is ToString to match JS naming ?

Reply:
Yes, `ToString` is a method of the object interface in JS.